File: Beecrowd/1002/1002.c
#include <stdio.h>

int main() {
    double raio, area;
    const double pi = 3.14159;

    scanf("%lf", &raio);
    area = pi * raio * raio;

    printf("A=%.4lf\n", area);

    return 0;
}

File: Beecrowd/1002/1002.cs
using System;

class URI
{

    static void Main(string[] args)
    {

        double raio = Convert.ToDouble(Console.ReadLine());

        double pi = 3.14159;

        double area = Math.Pow(raio, 2) * pi;

        Console.WriteLine($"A={area:F4}");
    }
}